molo/core/molo_wagtail_models.py

```diff
@@ -1,24 +0,0 @@
-from wagtail.core.models import Page
-from wagtail.core.utils import WAGTAIL_APPEND_SLASH
-from django.urls import reverse
-
-
-class MoloPage(Page):
-    class Meta:
-        proxy = True
-
-    def get_url_parts(self, request=None):
```

Comment:
Is this method now available as part of Page?

Reply:
yes